bitnami/jupyterhub/templates/hub/pdb.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.hub.pdb.enabled -}}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: hub
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  name: {{ template "common.names.fullname" . }}-hub
+spec:
+  minAvailable: {{ .Values.hub.pdb.minAvailable }}
```

Comment:
"maxAvailable" should be configurable too

Reply:
Done